product_docs/docs/efm/4/efm_rel_notes/index.mdx

```diff
@@ -1,14 +1,15 @@
 ---
 title: "Release notes"
 ---
-The Failover Manager documentation describes the latest version of Failover Manager 4, 
-including minor releases and patches. These release notes 
-cover what was new in each release. For new functionality introduced 
-in a minor or patch release, there are also indicators in the content 
+The Failover Manager documentation describes the latest version of Failover Manager 4,
+including minor releases and patches. These release notes
+cover what was new in each release. For new functionality introduced
+in a minor or patch release, there are also indicators in the content
 about the release that introduced the feature.
 
-| Version | Release Date | 
-| ------- | ------------ | 
+| Version | Release Date |
+| ------- | ------------ |
```

Comment:
Ugh, the above are whitespace only changes that I can't get my IDE to not make. Hopefully we don't actually need a blank space at the end of those lines.